content/foundations/color.mdx

```diff
@@ -17,21 +17,30 @@ import {Box, Button, Heading, Label, LabelGroup, Link} from '@primer/react'
   src="https://user-images.githubusercontent.com/6951037/146927448-cb518377-114a-4ab8-a37e-1eeffc8732c7.png"
 />
 
-GitHub's UI offers a variety of different color modes. When designing product interfaces, you should design in light mode by default. Every pattern in Primer is built to work across all color modes out of the box.
+
+GitHub's UI offers a variety of different color modes. Every pattern in Primer is built to work across all color modes out of the box.
+
+When designing product interfaces in Figma, we recommend to design in light mode. This is best, because the primer components for Figma are only available in light mode.
+To preview your work in other modes, use the [Figma color mode plugin](https://www.figma.com/community/plugin/992128487074360945/Change-Color-Mode).
+
+## How to use color for product UI
+
+Primer delivers colors in the form of [design tokens](https://primer.style/primitives/colors). Design tokens are a layer of abstraction that allows better maintainability, consistancy and easy theming.
+
+For example you use `bg-default` for the background of the page and `fg-default` for the text color. If the user changes to a dark mode, we change the underlying color that those tokens reference, but the tokens stay the same. 
```

Comment:
```suggestion
For example, if a page uses `bg-default` for the background color and `fg-default` for the text color, when the color mode is changed from light to dark, the tokens remain the same while their values update to respect the color mode. 
```

Reply:
Same as above. 😢 Love the change, but 9→14

Reply:
This one is not so easy. I think we can't just put a fullstop in there. @langermank any idea of how to make it work with two sentences?

Reply:
```suggestion
For example use `bg-default` for the background of the page and `fg-default` for the text color. If the user changes to a dark mode, the underlying color that those tokens reference change, but the tokens stay the same. 
```

Reply:
Love it. 👍